tests/kola/rpm-ostree/replace-rt-kernel

```diff
@@ -38,6 +38,10 @@ runv sed -i 's|^gpgkey.*|gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Offic
 case "${AUTOPKGTEST_REBOOT_MARK:-}" in
 "")
     echo "Testing overriding with CentOS Stream kernel"
+    # Disable all repos except baseos and appstream as not all of them have support for all RHCOS supported architectures
+    runv sed -i 's/enabled\ =\ 1/enabled\ =\ 0/g' /etc/yum.repos.d/cs.repo
```

Comment:
Are we assuming there are spaces in there? `enabled = 0`? I don't see any spaces in https://raw.githubusercontent.com/openshift/os/master/c9s.repo

Reply:
that is embarrassing. Fixing.